src/shared/ui/components/TransactionsTable.vue

```diff
@@ -0,0 +1,216 @@
+<script setup lang="ts">
+import { defaultFormat } from '@/shared/lib/time';
+import BaseTable from '@/shared/ui/components/BaseTable.vue';
+import BaseHash from '@/shared/ui/components/BaseHash.vue';
+import TransactionStatus from '@/entities/transaction/TransactionStatus.vue';
+import BaseLink from '@/shared/ui/components/BaseLink.vue';
+import type { useTable } from '@/shared/lib/table';
+import type { AccountId, Transaction } from '@/shared/api/schemas';
+import { computed } from 'vue';
+
+const props = withDefaults(
+  defineProps<{
+    table: ReturnType<typeof useTable<Transaction>>
+    filterBy?: { kind: 'authority', id: AccountId } | { kind: 'block', block: number } | null
```

Comment:
Replace with bool props that has direct meaning in this component:

- `show-block`
- `show-authority`